mecommerce/return-process
# fix(CorrectNameEventHeap): update events name of ReturnNotAutomated for heap to RefundNotAutomated

# Overview of the changes
Change the name of the Heap event ReturnNotAutomated to RefundNotAutomated.

### Jira Link
https://thirdlove.atlassian.net/browse/CXL-4

# Steps for testing
1. deploy the related branch on your testing env for dis-braze and process-return.
2. Process an auto refund and check that the new event RefundNotAutomated is reflecting on Heap.
